LinqToDBPostGisNetTopologySuite.Tests/GeometryAccessorsTests.cs

```diff
@@ -918,27 +900,41 @@ public void TestSTStartPoint()
         {
             using (var db = new PostGisTestDataConnection(TestDatabaseConnectionString))
             {
-                const string Wkt1 = "LINESTRING(0 1, 0 2)";
-                db.TestGeometries.Value(g => g.Id, 1).Value(g => g.Geometry, () => GeometryInput.STGeomFromText(Wkt1)).Insert();
+                const string wkt1 = "LINESTRING(0 1, 0 2)";
```

Comment:
The recommended naming and capitalization convention is to use [PascalCasing](https://en.wikipedia.org/wiki/Camel_case) for constants